.github/workflows/ci-additional.yml

```
@@ -0,0 +1,77 @@
+name: CI Additional
+
+on: [push, pull_request]
+
+jobs:
+  mindeps:
+    runs-on: "ubuntu-latest"
+    if: "contains(github.event.head_commit.message, 'test-mindeps') || github.event_name != 'pull_request'"
+
+    strategy:
+      matrix:
+        environment: ["mindeps-array-dataframe", "mindeps-bag-delayed"]
+
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+
+      - name: Setup Conda Environment
+        uses: goanpeca/setup-miniconda@v1.7.0
```

```suggestion
        uses: conda-incubator/setup-miniconda@v1.7.0
```

`goanpeca` migrated their action to the `conda-incubator` org